Monika After Story/game/script-holidays.rpy

```diff
@@ -6004,8 +6004,20 @@ label mas_f14_autoload_check:
         if not persistent._mas_f14_in_f14_mode and mas_isMoniNormal(higher=True):
             persistent._mas_f14_in_f14_mode = True
 
-            #Force sundress
-            monika_chr.change_clothes(mas_clothes_sundress_white, by_user=False, outfit_mode=True)
+            has_sundress = mas_SELisUnlocked(mas_clothes_sundress_white)
+
+            lingerie_eligible = (
+                mas_canShowRisque()
+                and not mas_SELisUnlocked(mas_clothes_vday_lingerie)
+                and has_sundress
+            )
+
+            if not has_sundress or (mas_SELisUnlocked(mas_clothes_blackpink_dress) and renpy.random.randint(1,2) == 1) or lingerie_eligible:
```

Comment:
I just made it slightly cleaner. Also why are we checking `lingerie_eligible` here?
```suggestion
            if (
                not has_sundress
                or (has_shoulderless and random.random() > 0.5)
                or lingerie_eligible
            ):
```

Reply:
because we don't want to give someone the black/pink dress the same day we give them lingerie